libsplinter/src/rest_api/auth/authorization/allow_keys.rs

```diff
@@ -40,21 +40,16 @@ use super::{AuthorizationHandler, AuthorizationHandlerResult};
 /// The list of keys in the file are cached in-memory by the authorization handler; this means that
 /// the handler will not have to read from the file every time permissions are checked. Instead,
 /// each time the handler checks for permissions, it will check the backing file for any changes
-/// since the last read, refreshing the internal cache if necessary. If the backing file is removed
```

Comment:
The commit message has some typos. I think it should say 

```
The backing file is considered a configuration file
rather than a state/data file, so the splinter node should not modify it in
any way (the node may not even have permissions to do so).
```

Reply:
Yep, double typo. Should be good now.